src/style_manager/index.ts

```diff
@@ -597,27 +597,65 @@ export default class StyleManager extends ItemManagerModule<
       const cmp = target.toHTML ? target : target.getComponent();
       const optsSel = { array: true } as const;
       let cmpRules: CssRule[] = [];
+      let tagNameRules: CssRule[] = [];
+      let invisibleRules: CssRule[] = [];
       let otherRules: CssRule[] = [];
       let rules: CssRule[] = [];
 
       const rulesBySelectors = (values: string[]) => {
-        return !values.length
-          ? []
-          : cssC.getRules().filter(rule => {
-              const rSels = rule.getSelectors().map(s => s.getFullName());
-              return !!rSels.length && rSels.every(rSel => values.indexOf(rSel) >= 0);
+        return cssC.getRules().filter(rule => {
+          const rSels = rule.getSelectors().map(s => s.getFullName());
+
+          // rSels is equal to 0 when rule selectors contain tagName like : p {}, .logo path {}, ul li {}
+          if (rSels.length === 0) {
+            return false;
+          }
+
+          return rSels.every(rSel => values.indexOf(rSel) >= 0);
+        });
+      };
+
+      const rulesByTagName = (tagName: string) => {
+        return cssC.getRules().filter(rule => {
+          const ruleTags = rule
+            .selectorsToString()
+            .split(' ')
+            .filter(item => {
+              return item.startsWith('.') === false && item.startsWith('#') === false;
             });
+
+          if (ruleTags.length === 0) {
+            return false;
+          }
+
+          const lastTags = ruleTags[ruleTags.length - 1];
+
+          return lastTags === tagName;
+        });
       };
 
       // Componente related rule
       if (cmp) {
         cmpRules = cssC.getRules(`#${cmp.getId()}`);
+        tagNameRules = rulesByTagName(cmp.get('tagName'));
         otherRules = sel ? rulesBySelectors(sel.getSelectors().getFullName(optsSel)) : [];
-        rules = otherRules.concat(cmpRules);
+        rules = otherRules.concat(tagNameRules).concat(cmpRules);
       } else {
         cmpRules = sel ? cssC.getRules(`#${sel.getId()}`) : [];
+        tagNameRules = rulesByTagName(sel?.get('tagName') || '');
```

Comment:
Keep in mind to avoid useless loops when it's possible

Reply:
Let's place a check 
```js
const rulesByTagName = (tagName: string) => {
        return !tagName ? [] : cssC.getRules().filter(rule => {
        ...
```

Reply:
I add the check 👍